qCC/ccClippingBoxTool.cpp

```diff
@@ -530,21 +532,31 @@ static unsigned ComputeGridDimensions(	const ccBBox& localBox,
 	return cellCount;
 }
 
+static constexpr char* s_originEntityUUID = "OriginEntityUUID";
```

Comment:
For modern C++, statics are replaced by using in an anonymous namespace (preferably at the top of the file). Things in anonymous namespaces are only valid within its translation unit.